windows/lib/WinPlatform.js

```diff
@@ -254,9 +254,10 @@ function(configId, args, callback) {
         version: manifest.appVersion + versionPadding,
         is_64_bit: true,
         icon: this.selectIcon(),
-        product: manifest.packageId
-        //extensions: 'tests/extension/echo_extension'
+        product: manifest.packageId,
+        extensions: manifest.extensions
```

Comment:
You can access the manifest inside WixSDK by using this.manifest, passing it like this is deprecated.